autoload/battery.vim

```diff
@@ -93,10 +93,11 @@ function! s:get_available_backend() abort
     return 'powershell'
   elseif battery#backend#linux#is_available()
     return 'linux'
+	elseif battery#backend#termux#is_available()
```

Comment:
Please fix the indent level.

Reply:
I didn't change the indent level tho, what do you mean?